server/server_test.go

```diff
@@ -58,3 +60,29 @@ var _ = Describe("AbsoluteURL", func() {
 		})
 	})
 })
+
+var _ = Describe("updateSocketPermission", func() {
+	tempDir, _ := os.MkdirTemp("", "")
+	socketPath := filepath.Join(tempDir, "test.sock")
+	file, _ := os.Create(socketPath)
+	file.Close()
```

Comment:
This should be in a BeforeEach block, and the temp dir should be deleted after each test.